app.py

```diff
@@ -79,4 +80,12 @@ def send_email():
 
 
 if __name__ == '__main__':
-    app.run()
+    host_name = '127.0.0.1'
```

Comment:
Should initialized value of host_name  be '0.0.0.0' ?

Reply:
Now the server config is `app.run(host = '0.0.0.0', port = 5000)`

Reply:
All right